code/_onclick/telekinesis.dm

```diff
@@ -139,6 +139,12 @@
 
 
 	else
+		if(focus.buckled_mobs)
+			to_chat(user, "<span class='notice'>This object is too heavy to move with something buckled to it!</span>")
+			return
+		if(length(focus.client_mobs_in_contents))
```

Comment:
Did you make sure this worked locally?

Reply:
This prevent TK if a player is in it yes, however it will not prevent if say a ckeyless mob is in it, like a monkey or xenobio slime. Unsure if you want that or not.

Reply:
This is fine.